<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Frank Osterfeld</author>
 <class>Akregator::SettingsAppearance</class>
 <widget class="QWidget" name="Akregator::SettingsAppearance">
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Article List Colors</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_unreadArticles">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Unread articles:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KColorButton" name="kcfg_ColorUnreadArticles">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_newArticles">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>New articles:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KColorButton" name="kcfg_ColorNewArticles">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_UseCustomColors">
        <property name="text">
         <string>Use custom colors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Font Size</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_MinimumFontSize">
        <property name="text">
         <string>Minimum font size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="slider_minimumFontSize">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPluralHandlingSpinBox" name="kcfg_MinimumFontSize">
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_MediumFontSize">
        <property name="text">
         <string>Medium font size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="slider_mediumFontSize">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPluralHandlingSpinBox" name="kcfg_MediumFontSize">
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fonts</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Standard font:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KFontComboBox" name="kcfg_StandardFont"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>Fixed font:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KFontComboBox" name="kcfg_FixedFont"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Serif font:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KFontComboBox" name="kcfg_SerifFont"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Sans serif font:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KFontComboBox" name="kcfg_SansSerifFont"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UnderlineLinks">
     <property name="text">
      <string>&amp;Underline links</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>KPluralHandlingSpinBox</header>
  </customwidget>
  <customwidget>
   <class>KFontComboBox</class>
   <extends>KComboBox</extends>
   <header>kfontcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
